src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { Comment } from '../../Types/Comment';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+type Props = {
+  comments: Comment[] | null,
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="comments-list" data-cy="comments-list">
+    <h3 className="comments-list__title">Comments:</h3>
+    {
+      comments
```

Comment:
Should check comments length (when comments are empty you don’t show 'No comments' - array always converts to true)